docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -73,15 +79,16 @@ ifdef::context[:parent-context: {context}]
 
 // Purpose statement for the assembly
 [role="_abstract"]
-As a developer of applications and services, you can use {product-long-api-designer} to ...
-{product-long-api-designer} is a managed cloud service that enables you to design schema and API definitions without having to install, configure, run, and maintain your own {product-api-designer} clusters.
+As a developer of applications and services, you can use {product-long-api-designer} to design standard event schemas and REST API definitions that comply with vendor-neutral and portable open specifications.
```

Comment:
Probably don't need "standard" here because we then have "that comply with vendor-neutral and portable open specifications"?